src/main/haskell/kore/src/Kore/Attribute/Trusted.hs

```diff
@@ -0,0 +1,60 @@
+{-|
+Module      : Kore.Attribute.Trusted
+Description : Trusted claim sentence attribute
```

Comment:
How about "Attribute identifying trusted claim sentences"?

Regardless of whether you like my suggestion or not, could explain in a few words what is a "trusted claim"?